packages/trader/src/Modules/MT5/Components/mt5-account-card.jsx

```diff
@@ -34,7 +35,7 @@ const MT5AccountCard = ({
     const has_demo_banner = type.category === 'demo';
 
     return (
-        <div className='mt5-account-card'>
+        <div className={`mt5-account-card ${!is_logged_in ? 'mt5-account-card__logged-out' : ''}`}>
```

Comment:
You can use `classNames()` here else there'll be an extra space (+ it will be consistent with our other conditional classes)

Reply:
noted on this...just realised there was a classNames() class by itself to handle this situation